crates/transpiler/src/passes/consolidate_blocks.rs

```diff
@@ -168,15 +169,28 @@ fn py_run_consolidate_blocks(
     blocks: Option<Vec<Vec<usize>>>,
     runs: Option<Vec<Vec<usize>>>,
 ) -> PyResult<()> {
+    // The node indices that enter from `blocks` and `runs` come from Python space, and we can't
+    // trust that they come from a correct analysis (or the block/run collection might have been
+    // invalidated).  Rather than panicking, we should raise Python-space exceptions.  We don't have
```

Comment:
Do we need these double spaces?

```suggestion
    // invalidated). Rather than panicking, we should raise Python-space exceptions. We don't have
```

Reply:
we don't, but good luck teaching my fingers that